app.js

```diff
@@ -0,0 +1,90 @@
+var app = {
+    Models: {},
+    Collections: {},
+    Views: {}
+};
+
+app.Models.TodoItem = Backbone.Model.extend({
+    defaults: {
+        title: undefined,
+        completed: false
+    }
+});
+
+app.Collections.TodoItems = Backbone.Collection.extend({
+    model: app.Models.TodoItem,
+    // localStorage: new Store("backbone-todo")
+});
+
+app.Views.TodoAppView = Backbone.View.extend({
+    el: ".todo",
+    initialize: function() {
+        
+        const todoItemsCollection = new app.Collections.TodoItems([
+            {title: "AAA"},
+            {title: "BBB"},
+            {title: "CCC"}
+        ]);
+        this.todoListView = new app.Views.ToDoListView({collection: todoItemsCollection});
+        this.subviews = [todoListView];
+
+        this.render();
+    },
+    template: Handlebars.compile($("#todo-app-template").html()),
```

Comment:
convention is that any of these class props go at the top of scope.

Reply:
> go at the top of the scope

Not sure what you mean exactly. Should it be first property declared in the literal? Is that what you mean?

Reply:
I mean this should go at the top with the other class props, above the initialize and any other functions.